cmd/kube-controller-manager/app/options/options.go

```diff
@@ -146,5 +150,9 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.RootCAFile, "root-ca-file", s.RootCAFile, "If set, this root certificate authority will be included in service account's token secret. This must be a valid PEM-encoded CA bundle.")
 	fs.Float32Var(&s.KubeAPIQPS, "kube-api-qps", s.KubeAPIQPS, "QPS to use while talking with kubernetes apiserver")
 	fs.IntVar(&s.KubeAPIBurst, "kube-api-burst", s.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver")
+	fs.DurationVar(&s.ClusterDrainTime.Duration, "cluster-drain-time", s.ClusterDrainTime.Duration,
+		"Time which should NodeController take to drain the whole cluster. See also max-node-eviction-period.")
+	fs.DurationVar(&s.MaxNodeEvictionPeriod.Duration, "max-node-eviction-period", s.MaxNodeEvictionPeriod.Duration,
```

Comment:
I don't like "max-node-eviction-period" as a flag name. I think these flags need a common prefix so they sort next to each other in help.